capstone/src/main/java/com/google/univiz/api/CollegeId.java

```diff
@@ -0,0 +1,7 @@
+package com.google.univiz.api;
+
+/**
+ * The CollegeId class will contain a college name and a unique college ID that can be used to
+ * identify the college
```

Comment:
Here is a suggestion to show you how you can document the behavior without actually leaking out implementation details.

```suggestion
 * This data structure contains all the college fields that are needed to uniquely identify a college. If two
 *  colleges have the same id, they they are the same college. Furthermore this structure may contain
 *  other alternative fields that could be used to uniquely identify a college.
```